net/sched/act_skbedit.c

```diff
@@ -53,6 +80,10 @@ static int tcf_skbedit(struct sk_buff *skb, const struct tc_action *a,
 
 	spin_unlock(&d->tcf_lock);
 	return d->tcf_action;
+
+err:
+	spin_unlock(&d->tcf_lock);
+	return TC_ACT_SHOT;
 }
 
 static const struct nla_policy skbedit_policy[TCA_SKBEDIT_MAX + 1] = {
```

Comment:
Add `[TCA_SKBEDIT_FLAGS] = { .len = sizeof(u64) },` to address Marcelo's feedback: https://www.mail-archive.com/netdev@vger.kernel.org/msg238536.html

Notice that you need to define `TCA_SKBEDIT_FLAGS` in `include/uapi/linux/tc_act/tc_skbedit.h`.

This way, the flags will be optional and won't break old versions of `tc`.